modules/zero-trust-manager-about-agent.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="zero-trust-manager-about-agent_{context}"]
+= SPIRE agent
+
+
+The SPIRE agent acts as a secure intermediary between the SPIRE server and the workloads running on a node. It performs node attestation against the server as well as workload attestation againt applications running on the node. The SPIRE agent is also responsible for retrieving, delivering, and renewing SVIDs. A SPIRE agent runs on every node on which an identified workload runs. For more information, see link:https://spiffe.io/docs/latest/spire-about/spire-concepts/#all-about-the-agent[SPIRE Agent].
```

Comment:
SPIRE agent's job is to primarily perform workload attestation when a Workload asks for an identity from the agent via the SPIFFE Workload API https://github.com/spiffe/spire/blob/v1.10.4/doc/plugin_agent_workloadattestor_k8s.md#k8s-selectors, it uses the Workload attestor plugin configured to perform the attestation. Incase of K8s its the K8s workload attestor plugin https://github.com/spiffe/spire/blob/v1.10.4/doc/plugin_agent_workloadattestor_k8s.md#k8s-selectors
Agent verification process is Node attestation during which the agent and server together verify the identity of the node the agent is running on.  Add links to the Spire concept doc https://spiffe.io/docs/latest/spire-about/spire-concepts/#node-attestation